ts/storage.ts

```diff
@@ -0,0 +1,20 @@
+import { TypeTodo } from "./demoTodo.js";
+
+const storage = {
+  get: function (key: string) {
+    const keyItem = localStorage.getItem(key);
+    if (keyItem) {
+      let value: string = JSON.parse(keyItem);
+      return value;
+    }
+  },
+
+  set: function (key: string, value: string | TypeTodo) {
```

Comment:
Why does this setter knows anything about todo ? `storage` is low abstraction module, meaning it shouldn't know anything about your business logic